### PR TITLE
Change after rubocop review

### PR DESCRIPTION
**Pull request by wolferbium**

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
-	private
-
-	def post_params
-		params.require(:post).permit(:title, :body)
-	end
-
-	def find_post
-		@post = Post.find(params[:id])
-	end
+  before_action :find_post, only: [:edit, :update, :show, :destroy]
```
Comment (AlxGolubev) [inline app/controllers/posts_controller.rb]: http://craftingruby.com/posts/2015/05/31/dont-use-before-action-to-load-data.html

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
-	private
+  private
 
-	def user_params
-		if current_user.admin
-			params.require(:user).permit(:email, :admin)
-		end
-	end
+  def user_params
+    params.require(:user).permit(:email, :admin) if current_user.admin
```
Comment (AlxGolubev) [inline app/controllers/users_controller.rb]: Please explain condition here, I don't understand why do you need it?

File: Gemfile [hunk 3]
```diff
@@ -39,20 +38,22 @@ group :development, :test do
...
 gem 'bootstrap-sass', '~> 3.3'
+gem 'simple_form', '~> 3.2'
 
 gem "haml-rails", "~> 1.0"
 
-gem "devise"
\ No newline at end of file
+gem "devise"
+
+gem 'rubocop', require: false
```
[the remark this answers is not in the document]
Comment (wolferbium) [reply Gemfile]: I've added it to .gitignore. Should it be added?

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (wolferbium) [reply app/controllers/users_controller.rb]: For changing email and admin state of definite user (as admin).

File: Gemfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (AlxGolubev) [reply Gemfile]: Yep

File: lib/application_responder.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class ApplicationResponder < ActionController::Responder
```
Comment (AlxGolubev) [inline lib/application_responder.rb]: You don't need your own responder, I'm almost sure, that default responders functionality will be enough

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,9 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+admin = User.new
```
Comment (AlxGolubev) [inline db/seeds.rb]: admin.admin = true

**Pull request merged**